src/tests/Transactions.test.js

```diff
@@ -0,0 +1,36 @@
+import { fireEvent, render, waitFor } from "@testing-library/react";
+import { getAllCheckingAccounts } from "../api";
+import { Transaction } from "../components/Transaction";
+
+const accountId = 150;
+const transactionAmount = 1200;
+const resolvedValue = {
+  data: [
+    {
+      id: 150,
+      name: "testing",
+    },
+  ],
+};
+
+jest.mock("../api/index");
+
+test("Should make a transaction in the selected account", async () => {
+  getAllCheckingAccounts.mockResolvedValue(resolvedValue);
+  const view = render(<Transaction />);
```

Comment:
Did you try passing the `type` to have a test per transaction (one for deposits and one for withdrawals)?

Reply:
That's a good idea